### DISCUSSION TITLE
Double stranded assembly graph

### DISCUSSION THREAD

**Discussion by ods1108** [ablab/spades#973] [General]
Hello 
I have a question abuot doble stranded assembly graph.
If we make all canonical(small lexicographic) we can save memory space half.
![image](https://user-images.githubusercontent.com/67948878/168411266-53a2ee1b-9aaa-4e14-b786-3676651e3282.png)
I think it make double memory usage of SPAdes.

**Comment by asl** [comment]
Hello

Thank you for your comment. Certainly noone stores sequences twice ;)
